flash-card.js

```diff
@@ -69,22 +85,64 @@ export default class FlashCardApp extends React.Component {
     for (var pair of formData.entries()) {
       cardObj[pair[0]] = pair[1]
     }
-    const flashCardStateCopy = this.state.flashcards.slice(0)
-    flashCardStateCopy.splice(editIndex, 1, cardObj)
+    const flashcardsCopy = this.state.flashcards.slice(0)
+    flashcardsCopy.splice(editIndex, 1, cardObj)
+    const updatedTopics= []
+    flashcardsCopy.map(flashcard =>  {
```

Comment:
Try using `Array.prototype.filter` instead of `map`. Avoid using `map` to just loop through an array. That's what `forEach` is for. If you get stuck trying to use `filter`, you can use `forEach`.